remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -453,7 +453,7 @@ public void testAddTags()
         List<Tag> tags = tagNames.stream().map(name -> Tag.builder().tag(name).create()).collect(toList());
         given(taggingServiceMock.addTags(CONTENT_NODE_REF, tagNames)).willReturn(pairs);
 
```

Comment:
given(parametersMock.getInclude()).willReturn(List.of("count")); should be added to initialize the mock value for count.

Reply:
Done